intelmq-mailgen

```diff
@@ -206,33 +353,32 @@ def send_notifications(config, notifications, cur):
     :param config script configuration
     :param notifications a list of notifications
     :param cur database cursor to use when loading event information
+
+    :returns: number of send mails
+    :rtype: int
     """
+    send_mails = 0
```

Comment:
The variable "send_mails" is used to keep track of how many mails have been sent, so it should be spelled "sent_mails" ('t' instead of 'd'). In the caller as well.